starlight_help/src/content/docs/protect-your-account.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Protect your account
+---
+
+import {Steps} from "@astrojs/starlight/components";
+import ZulipTip from '../../components/ZulipTip.astro';
+
+import FlattenedSteps from "../../components/FlattenedSteps.astro";
+import NavigationSteps from "../../components/NavigationSteps.astro";
+import ZulipNote from "../../components/ZulipNote.astro";
+
+Every Zulip user has an API key, which can be used to do essentially everything
+that you can do when you're logged in. You should treat your Zulip API key as
+carefully as you treat your password.
+
+Be sure to reset both your password and API key immediately in the following
+situations:
+
+- Accidentally sharing your password or API key with someone else.
+- Losing a device where you're logged in to Zulip (e.g., a cell phone or laptop).
+- Losing a device that has a downloaded copy of your [zuliprc
+  file](/api/configuring-python-bindings#download-a-zuliprc-file), which
+  includes your API key.
+
+Resetting your API key will immediately log you out of Zulip on all devices.
+
+## Reset your password and API key
+
+<FlattenedSteps>
+  <NavigationSteps target="settings/account-and-privacy" />
+
+  1. Under **Account**, click **Change your password**.
+  1. Enter your old password and your new password, and click **Change**.
+  1. Under **API key**, click **Manage your API key**.
+  1. Enter your new password, and click **Get API key**.
+  1. Click **Generate new API key**.
+</FlattenedSteps>
+
+<ZulipTip>
+    If you've forgotten your password, you can [reset
+    it](/help/change-your-password#if-youve-forgotten-or-never-had-a-password)
+    using your email account.
+</ZulipTip>
+
```

Comment:
I think we want another tip here suggesting that you should only use your API key for integrations that need to act as your account; most integrations should use a bot user, which doesn't expose your account and organization to as much risk if the bot's API key is stolen.